crown-apparel/cloud-config.yaml

```diff
@@ -4,5 +4,5 @@ users:
     sudo: ALL=(ALL) NOPASSWD:ALL
     shell: /bin/bash
     ssh_authorized_keys:
-      - ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIOuuQ7+MWvGOpJoW4NUntltko6o92VgLCjsEp696VpdM example@example.com
```

Comment:
Everytime we run your automation, we're going to change this file and have it in the code changes?